isis/tests/FunctionalTestsLronacpho.cpp

```diff
@@ -0,0 +1,134 @@
+#include <QTemporaryDir>
+#include "LineManager.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Endian.h"
+#include "PixelType.h"
+
+#include "lronacpho.h"
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using namespace testing; 
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/lronacpho.xml").expanded();
+
+TEST(LronacphoNoDemAlgo3, FunctionalTestsLronacpho) {
+  QTemporaryDir prefix;
+
+  ASSERT_TRUE(prefix.isValid());
+
+  QString inCubeFileName = "data/lronacpho/M143947267L.cal.echo.crop.cub";
```

Comment:
Assuming this image is small enough (like single digit MB), you can just check it in as part of this PR.

Reply:
It is not a single-digit MB. It is 8.4 MB

Reply:
I'll crop down to 1 MB and add to PR

Reply:
Added test cube